lib/rubocop/cop/rspec/message_spies.rb

```diff
@@ -36,7 +36,7 @@ class MessageSpies < Cop
         SUPPORTED_STYLES = %w(have_received receive).freeze
 
         def_node_matcher :message_expectation, %(
-          (send (send nil :expect (send nil $_)) :to ...)
+          (send (send nil :expect {({send const} nil $_) ({ivar lvar} $_)}) {:to :to_not :not_to} ...)
```

Comment:
Finally I understood how to use RuboCop::Node.
(catch all by `$(...)` and return to original code by `RuboCop::Node#source` )
thanks!